source/postcard/Cargo.toml

```diff
@@ -66,6 +66,11 @@ optional = true
 version = "1.0.12"
 optional = true
 
+[dependencies.nalgebra]
+version = "0.33.0"
```

Comment:
Since nalgebra isn't 1.0, could you please make this an explicit feature with a name like `nalgebra-0_33`? This will prevent this from being an issue in the future if we need to support multiple versions.

Reply:
Done